server.py

```diff
@@ -3,13 +3,13 @@
 inputs = []
 outputs = []
 message_queues = {}
-
+data_to_send = 0
 
 def add_new_client(new_socket):
     connection, client_address = new_socket.accept()
     connection.setblocking(0)
     inputs.append(connection)
-    message_queues[connection] = Queue.Queue()
+    #message_queues[connection] = Queue.Queue()
```

Comment:
Please don't leave commented code lines, you have source control so you can look at your old version

Reply:
fixed